### PR TITLE
state-card-sun

### PR DESCRIPTION
**Pull request by micw**
New sun state card that shows next sunrise and next sunset

### CODE REVIEW

File: src/state-summary/state-card-sun.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+<link rel="import" href="../components/entity/state-info.html">
+
+<dom-module id="state-card-sun">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      .card {
+	    @apply(--paper-font-body1);
```
Comment (balloob) [inline src/state-summary/state-card-sun.html]: This indentation is off

**Comment by balloob** [comment]
Can you include a screenshot?

State cards need to be able to work when the columns are very narrow too.

**Comment by Landrash** [comment]
If your reworking the sun card I would suggest going for a design similar to this to match the new weather card from #352 .
sun-card

**Comment by balloob** [comment]
This PR seems to have gone stale. Closing it. You can reopen it when you're ready to finish it.

**Pull request closed without merging**